backend/src/main/java/com/jobwait/persistence/PostgresController.java

```diff
@@ -156,7 +147,9 @@ public User createUserFromAuthId(String authHash) {
 
             // init null answers for user
             this.updateUserAnswers(user, new ArrayList<Answer>());
-
+                       
+            connection.close();
```

Comment:
I was not aware that it would crash but honestly in hindsight we should have known this lmao. Like every connection expects you to close it. Either way good thing we found out before proper prod